src/cpp/session/modules/SessionThemes.cpp

```diff
@@ -154,6 +154,12 @@ void getThemesInLocation(
                (std::istreambuf_iterator<char>()));
             themeIFStream.close();
 
+            // Skip theme if file is empty.
+            if (themeContents.empty())
+            {
+               continue;
+            }
```

Comment:
Why is the theme file empty?

Reply:
That's a great question.  I was trying to fix this warning:

```
 09 Feb 2021 18:13:29 [rsession-jmcphers] WARNING rs-theme-is-dark is not set for theme "eiffel".; LOGGED FROM: void rstudio::session::modules::themes::(anonymous namespace)::getThemesInLocation(const rstudio::core::FilePath &, const std::string &, rstudio::session::modules::themes::(anonymous namespace)::ThemeMap *) src/cpp/session/modules/SessionThemes.cpp:195
09 Feb 2021 18:13:29 [rsession-jmcphers] WARNING rs-theme-is-dark is not set for theme "eiffel".; LOGGED FROM: void rstudio::session::modules::themes::(anonymous namespace)::getThemesInLocation(const rstudio::core::FilePath &, const std::string &, rstudio::session::modules::themes::(anonymous namespace)::ThemeMap *) src/cpp/session/modules/SessionThemes.cpp:195
```

I could not figure out why this would be the case, so I pulled the Eiffel theme out of the test matrix, only to watch the error start happening for the Espresso Libre theme. Debugging revealed that `rs-theme-is-dark` was not set because the entire file was empty. I strongly suspect this has something to do with the alphabetically adjacent `empty.rstheme` file, but ignoring empty themes seems like the right thing to do anyway (not just for tests) ... and it also happens to make the warning go away.